CONTRIBUTING.md

```diff
@@ -181,15 +181,34 @@ However, it's not always that simple. Our [PR approval process](#pr-approval-pro
 
 One thing you will need to do only for your first pull request in Netdata is to accept the CLA. Until you do, the automated check for the CLA acceptance will be showing as failed. 
 
-#### PR title guidelines
+#### PR Guidelines
 
-All verbs in PR titles must be in the past tense. E.g. "Added" instead of "Adds", "Fixed" instead of "Fixes".
+PR Titles:
 
-Bug fixes must have the title "Fixed [description of the problem as a user would perceive it], by [explanation of the correction]". 
+- Must follow the [Imperative Mood](https://en.wikipedia.org/wiki/Imperative_mood)
```

Comment:
I don't agree, we'll need to replace all verbs when writing release notes. Do suggest having the release notes in the imperative mood as well?

Reply:
Change to their past tense?

@cosmix Do you have some comments here?

Reply:
I think an argument can be made for both past or present tense, so I won't get too involved in that. I don't think changing the tense for release notes will be _too_ much of an issue moving forward, as Patti and I are going to be updating the release notes in an ongoing basis as the sprints move along. It's a lot less daunting to do it piecemeal than in one giant batch like you did for v1.20, Chris. Just a thought.